Source/MonoGame.Extended/FramesPerSecondCounter.cs

```diff
@@ -4,9 +4,10 @@
 
 namespace MonoGame.Extended
 {
-    public class FramesPerSecondCounter : IUpdate
+    public class FramesPerSecondCounter : DrawableGameComponent
     {
-        public FramesPerSecondCounter(int maximumSamples = 100)
+        public FramesPerSecondCounter(Game game, int maximumSamples = 100)
+            :base(game)
```

Comment:
@craftworkgames  We can create a new `GameComponent` class which doesn't take in a `Game` as a parameter to be constructed.